sonar-csharp-plugin/pom.xml

```diff
@@ -147,6 +147,18 @@
       <version>${sonar.api.impl.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.sonarsource.api.plugin</groupId>
+      <artifactId>sonar-plugin-api-test-fixtures</artifactId>
+      <version>${sonar.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>${slf4j.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
This should be compile-time problem, that only breaks at runtime due to some very weird way how Java dependency hell works